docs/basic-docs/atomicdex-api-20-dev/enable_bch_with_tokens.md

```diff
@@ -0,0 +1,591 @@
+| bchd_urls              | string   | An array of strings. URLs of BCHD gRPC API servers that are used for SLP tokens transactions validation. It's recommended to add as many servers as possible. The URLs list can be found at https://bchd.fountainhead.cash/.          |
+| mode                   | string   | Utxo RPC mode. Options: `{ "rpc":"Native" }` if running a native blockchain, or `"rpc":"Electrum"` to use electrum RPCs. If using electrum, a list of electrum servers is required under `rpc_data.servers` |
+| tx_history             | boolean  | If `true`, spawns a background loop to store the local cache of address(es) transactions. Defaults to `false`. |
+| slp_tokens_requests    | string   | Array of SLP activation requests. SLP activation requests contain mandatory `ticker` and optional `required_confirmations` fields. If required_confirmations is not set for a token, then MM2 will use the confirmations setting from its coins config or platform coin. |
+| required_confirmations | integer  | Optional. Confirmations to wait for steps in swap. Defaults to value in the coins file if not set. |
+| requires_notarization  | boolean  | Optional. Has no effect on BCH. Defaults to `false`. |
+| address_format.format  | string   | Optional. Overwrites the address format from coins file, if set. Options: `{"format":"standard"}` for legacy/standard address format, `{"format":"cashaddress"}` for cash address format |
+| address_format.network | string   | Optional. Overwrites the address format from coins file, if set. Options: `{"format":"standard"}` for legacy/standard address format, `{"format":"cashaddress"}` for cash address format |
+| utxo_merge_params      | boolean  | Optional. If defined, will spawn a background loop that checks the number of UTXOs every `check_every seconds` and merges `max_merge_at_once` utxos into a single utxo if the total exceeds `merge_at`. Recommended for addresses with high trading activity or mining which can increase the number of UTXOs in the address, leading to delays in RPC response (or in extreme cases, connection timing out). |
+
+
+## Example using Electrum servers on testnet with tx_history, cashaddress format and automated utxo merging.
+
+
+```bash
+curl --url "http://127.0.0.1:7783" --data '{
+  "userpass":"'$userpass'",
+  "method":"enable_bch_with_tokens",
+  "mmrpc":"2.0",
+  "params":{
+    "ticker":"BCH",
+    "allow_slp_unsafe_conf":false,
+    "bchd_urls":[
+      "https://bchd.imaginary.cash:8335/"
+    ],
+    "mode":{
+      "rpc":"Electrum",
+      "rpc_data":{
+        "servers":[
+          {
+            "url":"electrum1.cipig.net:10055"
+          },
+          {
+            "url":"electrum2.cipig.net:10055"
+          },
+          {
+            "url":"electrum3.cipig.net:10055"
+          },
+          {
+            "url":"electrum1.cipig.net:20055"
+          },
+          {
+            "url":"electrum2.cipig.net:20055"
```

Comment:
20055 are SSL ports. The object should have `protocol:SSL` as per https://github.com/KomodoPlatform/coins/blob/master/electrums/BCH#L25